### PR TITLE
ADS-3448: Improve Dynamic Bundle documentation

### PR DESCRIPTION
**Pull request by manikandan-ravikumar**
This PR adds details about mandatory variables to be configured for proper functioning of Dynamic Bundle script and includes details about the differences between the hash & secret key and where they are used

## JIRA
[ADS-3447](https://nostosolutions.atlassian.net/browse/ADS-3447)

### CODE REVIEW

File: features/dynamic-bundle-discounts.md [hunk 1]
```diff
@@ -15,6 +16,20 @@ This documentation explains the process of setting up **Nosto - Dynamic Bundles*
 
 ### Nosto Dynamic Bundle template
 
+Nosto bundle templates needs the following mandatory variables (name intact) to be configured for proper functioning:
+1. discount type - Defines the type of discount offered (percent/dollar)
```
Comment (olsi-qose) [inline features/dynamic-bundle-discounts.md:20]: `percent/amount`  might be a better terminology

File: features/dynamic-bundle-discounts.md [hunk 2]
```diff
@@ -15,6 +16,20 @@ This documentation explains the process of setting up **Nosto - Dynamic Bundles*
...
+2. discount value - Defines the amount/percent of discount offered on products
+3. secured - A flag that can be used to specify if the merchant willing to use `hash` based authentication for bundle discounts
```
Comment (olsi-qose) [inline features/dynamic-bundle-discounts.md]: Better re-phrase it to: 
Enable `hash` based authentication for bundle discounts

File: features/dynamic-bundle-discounts.md [hunk 3]
```diff
@@ -107,7 +122,12 @@ Please follow the steps below for setting up the line item script for handling t
 \#end\
 \# ================================================================
 
-### Known Issues
+## Important Note
+Nosto Dynamic Bundle configuration involves two important keys, hash and secret key. Hash key is used within the bundle template and can be accessed using predefined variation `$hash`. Please avoid hard-coding this value anywhere inside the template. As dynamic bundle configurations are subjected to change, the hash key will also change accordingly. Hard-coding this key may break the functionality.
+
+Seret key, on the other hand, can be retrieved from "Dynamic Bundle Key" field in Nosto Admin Settings > Platform page. This value should be shared with merchants which then gets included in the Shopify line item script replacing the GET\_FROM\_NOSTO placeholder
```
Comment (olsi-qose) [inline features/dynamic-bundle-discounts.md]: Typo in secret*

File: features/dynamic-bundle-discounts.md (hunk 1 above)
Comment (manikandan-ravikumar) [reply features/dynamic-bundle-discounts.md:20]: `dollar` is used by shopify. `amount` not supported

File: features/dynamic-bundle-discounts.md (hunk 3 above)
Comment (manikandan-ravikumar) [reply features/dynamic-bundle-discounts.md]: Done

File: features/dynamic-bundle-discounts.md (hunk 2 above)
Comment (manikandan-ravikumar) [reply features/dynamic-bundle-discounts.md]: Done

**Review by olsi-qose — APPROVED** [review]

**Pull request merged**